lib/dotcom_web/plugs/path_params_redirector.ex

```diff
@@ -0,0 +1,34 @@
+defmodule DotcomWeb.Plugs.PathParamsRedirector do
+  import Plug.Conn, only: [put_status: 2, halt: 1]
+  import Phoenix.Controller, only: [redirect: 2]
+
+  @behaviour Plug
+
+  @impl Plug
+  @spec init(Keyword.t()) :: Keyword.t()
+  def init([to: _] = opts), do: opts
+  def init(_opts), do: raise("Missing required to: option in redirect")
+
+  @impl true
```

Comment:
This is a Plug implemented function too.

Reply:
[Yes it is - thanks for catching!](https://github.com/mbta/dotcom/pull/2432/commits/6fff69862b804b3dec8044389b566501372861d6)

Good thing I [also forgot to add a moduledoc](https://github.com/mbta/dotcom/pull/2432/commits/1457f18a4a4af995d0744608b31d8bf03095904b), so that this didn't get scooped by auto-merge 😅